loader/src/sources/albertsons/index.js

```diff
@@ -156,6 +157,16 @@ const BRANDS = [
     url: "https://www.safeway.com/pharmacy/covid-19.html",
     pattern: /Safeway/i,
   },
+  {
+    ...BASE_BRAND,
+    key: "sav_on",
+    // Sav-On pharmacies are sometimes referred to as Albertsons Pharmacies.
+    // If an adult listing is named one way and a pediatric location is named
+    // the other, they need both kinds of external IDs so we can merge them.
+    keys: ["albertsons", "sav_on"],
```

Comment:
This is ugly, but I did a check to see whether doing it for all brands would cause problems, and that goes wrong way more often than it goes right. :(

It seems to work for the case we’re hitting, although a more conservative approach might just be to add something to `corrections.js` for Sav-On #4706 (e.g. by re-naming it to Albertsons, or vice-versa).

Wikipedia makes me feel like my code here might be OK, but also the particular branding rules are complex (assuming what Wikipedians have written is even fully correct, and that the messy data we’re getting will reliably match those rules):

> The Osco and Sav-on brands survive today as brandings for in-store pharmacies in the Albertsons chain of stores. Osco pharmacies can be found in [Jewel](https://en.wikipedia.org/wiki/Jewel_(supermarket)) stores, which are co-branded Jewel-Osco, as well as in [Shaw's and Star Markets](https://en.wikipedia.org/wiki/Shaw%27s_and_Star_Market). Sav-on pharmacies are found largely in [Acme](https://en.wikipedia.org/wiki/Acme_Markets) stores. Pharmacies in Albertsons stores are called Osco in Arizona and Montana stores, and Sav-On in other markets, predominantly in Texas for example.
> (From https://en.wikipedia.org/wiki/Osco_Drug_and_Sav-on_Drugs)

```
"Albertsons 3195" -> Market Street 3195
  3499 E. Fariview Ave, Meridian, ID 83642
  3499 E Fairview Ave., Meridian, ID 83642

"Albertsons 3713" -> Pediatric COVID Clinic at Safeway Pharmacy on 3713
  15232 George O'Neal Rd., Baton Rouge, LA 70817
  3713 Lee Highway, Arlington, VA 22207

"Albertsons 505" -> United 505
  5415 S.W. Beav.-Hillsdale Hwy, Portland, OR 97221
  401 Slide Road, Lubbock, TX 79416

"Albertsons 515" -> United 515
  3013 N W Stewart Parkway, Roseburg, OR 97471
  311 South Ave D, Burkburnett, TX 76354

"Albertsons 561" -> Market Street 561
  298 E. Live Oak Ave., Arcadia, CA 91006
  6100 West Eldorado Parkway, McKinney, TX 75070

"Albertsons 574" -> United 574
  5755 S. Main Street, Springfield, OR 97478
  2160 Pine St., Abilene, TX 79601

"Albertsons Market 676" -> Safeway 676
  2751 County Road West, Odessa, TX 79764
  1500 Solano Ave, Albany, CA 94706

"Albertsons Market 678" -> Shaws 678
  3121 Sunset Drive, San Angelo, TX 76904
  7 Continental Blvd., Merrimack, NH 03054

"Albertsons Market 691" -> Safeway 691
  900 W. 2nd Street, Roswell, NM 88201
  1444 Shattuck Place, Berkeley, CA 94709

"Albertsons Market 920" -> Safeway 920
  710 A Paseo del Pueblo Sur, Taos, NM 87571
  451 East Wonderview Avenue, Estes Park, CO 80517

"Amigos 503" -> Safeway 503
  112 N. University, Lubbock, TX 79415
  11031 19th Ave S E, Everett, WA 98208

"Amigos 508" -> Safeway 508
  2403 N. Columbia Ave., Plainview, TX 79072
  401 A Avenue, Lake Oswego, OR 97034

"Market Street 502" -> Safeway 502
  3405 50th Street, Lubbock, TX 79413
  5702 Summitview Ave, Yakima, WA 98908

"Market Street 526" -> Safeway 526
  2530 S. Georgia, Amarillo, TX 79109
  14444 124th Ave N E, Kirkland, WA 98034

"Market Street 560" -> Albertsons 560
  5605 Colleyville Blvd., Colleyville, TX 76034
  55 Division Ave., Eugene, OR 97404

"Market Street 563" -> Safeway 563
  985 W. Bethany Dr., Allen, TX 75013
  613 Sixth, Sunnyside, WA 98944

"Market Street 564" -> Albertsons 564
  700 N. Denton Tap Rd., Coppell, TX 75019
  451 NE 181st Ave, Portland, OR 97230

"Safeway 521" -> United 521
  20151 Se Highway 212, Damascus, OR 97089
  2522 Iowa Park Rd., Wichita Falls, TX 76306

"Safeway 525" -> United 525
  1003 Medford Center, Medford, OR 97504
  2806 S. Main, Perryton, TX 79070

"Safeway 533" -> United 533
  19150 N E Woodinville Duvall, Woodinville, WA 98077
  3400 River Road, Amarillo, TX 79107

"Safeway 534" -> United 534
  3532 172nd St, Arlington, WA 98223
  1601 Tennessee, Dalhart, TX 79022

"Safeway 537" -> United 537
  19651 Hwy 2, Monroe, WA 98272
  2703 82nd Street, Lubbock, TX 79423

"Safeway 543" -> Market Street 543
  4700 Yelm Rd S E, Lacey, WA 98503
  4205 98th Street, Lubbock, TX 79423

"Safeway 548" -> United 548
  44428 Sterling Hwy S, Soldotna, AK 99669
  3301 South 14th, Abilene, TX 79605

"Safeway 549" -> United 549
  500 East Third Street, Alliance, NE 69301
  3500 College Avenue, Snyder, TX 79549

"Safeway 911" -> Albertsons Market 911
  477 W Napa St, Sonoma, CA 95476
  2801 A Eubank Blvd, Albuquerque, NM 87112

"Safeway 917" -> Albertsons Market 917
  600 S Broadway, Walnut Creek, CA 94596
  1625 Rio Bravo Blvd. S.W., Albuquerque, NM 87105

"United 509" -> Safeway 509
  511 College Ave, Levelland, TX 79336
  5920 NE MLK Blvd, Portland, OR 97211

"United 513" -> Albertsons 513
  2900 Wilbarger Ave, Vernon, TX 76384
  4740 Royal Avenue, Eugene, OR 97402

"United 514" -> Safeway 514
  1000 W. 11th St., Quanah, TX 79252
  642 Ne Third Street, Bend, OR 97701

"United 536" -> Albertsons 536
  1401 Tahoka Road/Rt. 1, Brownfield, TX 79316
  25691 SE Stark, Troutdale, OR 97060

"United 547" -> Safeway 547
  920 N. Willis, Abilene, TX 79603
  10105 224th St E, Graham, WA 98338

"United 552" -> Albertsons 552
  6313 4th Street, Lubbock, TX 79416
  27702 Crown Valley Parkway, Ladera Ranch, CA 92694

"United 555" -> Safeway 555
  12815 Indiana Ave, Lubbock, TX 79423
  630 228th Ave N E, Sammamish, WA 98074
```